docs/usage/config-presets.md

```diff
@@ -217,13 +217,12 @@ Create a [discussion](https://github.com/renovatebot/renovate/discussions) to pr
 The maintainers can also help improve the preset, and let you know where to put it in the code.
 If you are proposing a "monorepo" preset addition then it's OK to raise a PR directly as that can be more efficient than a GitHub Discussion.
 
-## Organization level presets
+## Group/Organization level presets
 
-Whenever repository onboarding happens, Renovate checks if the current user/group/org has a default config to extend.
-It looks for:
-
-- A repository called `renovate-config` under the same user/group/org with a `default.json` file or
-- A repository named like `.{{platform}}` (e.g. `.github`) under the same user/group/org with `renovate-config.json`
+Whenever repository onboarding happens, Renovate checks for a a default config to extend.
+Renovate will check for a repository called `renovate-config` with a `default.json` file in the parent user/group/org of the repository.
+On platforms that support nested groups (e.g. GitLab), Renovate will check for this repository at each level of grouping, from nearest to furthest, and use the first one it finds.
+On all platforms, it will also look for a repository named like `.{{platform}}` (e.g. `.github`) with a `renovate-config.json`, under the same top-level user/group/org.
```

Comment:
```suggestion
On all platforms, it will then look for a repository named like `.{{platform}}` (e.g. `.github`) with a `renovate-config.json`, under the same top-level user/group/org.
```